spec/system/admin/reports/orders_and_fulfillment_spec.rb

```diff
@@ -461,6 +458,28 @@
                 expect(page).not_to have_css("td.header-row")
               end
             end
+
+            xit "aggregates results per variant" do
+              pending '#9678'
+              expect(all('table.report__table tbody tr').count).to eq(4)
+              # 1 row per variant = 2 rows
+              # 2 TOTAL rows
+              # 4 rows total
+
+              expect(table_headers[0]).to eq(
+                ["Supplier Name", "Baked Beans", "1g Small, S",
+                 "Distributor Name", "7", "10.0", "70.0", "UPS Ground"]
+              )
+              expect(table_headers[1]).to eq(
+                ["", "", "", "TOTAL", "7", "", "70.0", ""]
+              )
+              expect(table_headers[2]).to eq(
+                ["Supplier Name", "Baked Beans", "1g Big, S",
+                 "Distributor Name",
+                 "3", "10.0", "30.0", "UPS Ground"]
+              )
+              expect(table_headers[3]).to eq(["", "", "", "TOTAL", "3", "", "30.0", ""])
+            end
```

Comment:
Is this correct? Issue https://github.com/openfoodfoundation/openfoodnetwork/issues/9678 is closed.

Reply:
Yes indeed - thanks @dacook. Inconsistent results ordering is now fixed and the spec is now updated.